store/sqlstore/supplier.go

```diff
@@ -1148,3 +1149,11 @@ func convertMySQLFullTextColumnsToPostgres(columnNames string) string {
 
 	return concatenatedColumnNames
 }
+
+func getQueryBuilder(s SqlStore) sq.StatementBuilderType {
```

Comment:
Let's define `getQueryBuilder` /on/ `SqlStore` instead to allow `s.getQueryBuilder()`:
```go
func (s SqlStore) getQueryBuilder() sq.StatementBuilderType {
      // ...
}
```

Reply:
Sounds good. Will do.